drivers/scaleway/provider.go

```diff
@@ -49,8 +49,7 @@ func New(opts ...Option) (autoscaler.Provider, error) {
 		p.size = "dev1-l"
 	}
 	if p.image == "" {
-		// Ubuntu  for dev1-m in fra-par-1
-		p.image = "f974feac-abae-4365-b988-8ec7d1cec10d"
+		p.image = "ubuntu-bionic"
```

Comment:
TIL this works. I didn't realise it would do the conversion for you. Thanks for looking into this

Reply:
It's possible that it had been added in a later version of the SDK, i had this information from the Scaleway slack channel. The create server function is using https://godoc.org/github.com/scaleway/scaleway-sdk-go/api/marketplace/v1#API.GetLocalImageIDByLabel internally, which matches the right UUID from the marketplace API.

You can fetch all available labels via `curl -s 'https://api-marketplace.scaleway.com/images?page=1&per_page=100' | jq .images[].label | tr '_' '-'`.